standalone/runtime/src/lib.rs

```diff
@@ -280,6 +280,11 @@ impl pallet_timestamp::Config for Runtime {
 	type WeightInfo = ();
 }
 
+#[allow(clippy::identity_op)]
+#[cfg(feature = "integration-tests")]
+pub const EXISTENTIAL_DEPOSIT: u128 = 1000;
+
+#[cfg(not(feature = "integration-tests"))]
```

Comment:
you did declare the `EXISTENTIAL_DEPOSIT` in `primitives/src/lib.rs` and you are redeclaring a different one here, why not using the one from the `primitives/src/lib.rs`?